rn/Teacher/index.ios.js

```diff
@@ -137,6 +140,12 @@ const loginHandler = async ({
   Helm.loginComplete()
   loginVerify()
   beginUpdatingBadgeCounts()
+
+  if (wasReload) {
+    let lastRoute = await getLastRoute()
+    let navigator = new Navigator('')
+    navigator.show(lastRoute.url, lastRoute.options, lastRoute.props)
```

Comment:
this will bring you back to the same screen, but it won't have the navigation stack below. Is that ok?

Reply:
Yeah, for now that works for me. If we want to try to get the whole navigation stack at some point we could investigate that.